examples/README.md

```diff
@@ -34,7 +34,19 @@
 - `static-files`
   - Learn how to serve static files.
   - See [Serving static files](https://line.github.io/armeria/server-http-file.html).
+  
+## How to start
+
+Enter the command below in the **Armeria root** directory.  
+`{EXAMPLE_NAME}` is a directory name.
+
+```bash
+# default
+./gradlew :examples:{EXAMPLE_NAME}:run
```

Comment:
The `examples` directory is copied into https://github.com/line/armeria-examples so that a user can play without building Armeria. Once copied, this information becomes incorrect because it is no more a multi-module project. Instead, could you rewrite this section in a more generic way, like 'Use `run` or `bootRun` task to run an example from Gradle.' ?

Also, could you mention that a user can run an example from an IDE by running the `Main` class of each example?

Reply:
>The examples directory is copied into https://github.com/line/armeria-examples so that a user can play without building Armeria. Once copied, this information becomes incorrect because it is no more a multi-module project. Instead, could you rewrite this section in a more generic way, like 'Use run or bootRun task to run an example from Gradle.' ?

Are you planned put `gradlew` file into each example directory?
(ex: `examples/grpc-service/gradlew`)
Because there is `gradlew` file in each copied example, but currently is not.

> Also, could you mention that a user can run an example from an IDE by running the Main class of each example?

Okay..!

Reply:
> Are you planned put `gradlew` file into each example directory?
(ex: `examples/grpc-service/gradlew`) Because there is `gradlew` file in each copied example, but currently is not.

It is copied for line/armeria-examples but not for this repository, so you'll have to write something in a generic manner, like I mentioned in the comment above. Please go look into line/armeria-example, which is not 'examples' in line/armeria.

Reply:
Understood, I updated files!